include/options/boolean.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Class to manage single boolean option.
+ * Note that for historic reason, boolean are stored as 0 or 1.
+ *
+ * @since 3.7
+ */
+class PLL_Boolean_Option extends PLL_Abstract_Option {
+	/**
+	 * Validates option's value,
+	 * only boolean or 0 and 1 accepted (int or string).
+	 *
+	 * @since 3.7
+	 *
+	 * @param mixed $value Value to validate.
+	 * @return bool True if the value is valid, false otherwise.
```

Comment:
I think this should prevent to trigger a PHPStan error for the param of `sanitize()`:
```suggestion
	 * @return bool True if the value is valid, false otherwise.
	 *
	 * @phpstan-assert-if-true bool|int|numeric-string $value
```